engines/icb/animation_mega_set.cpp

```diff
@@ -281,7 +281,7 @@ bool8 _vox_image::Init_custom_animation(const char *anim) {
 
 	// check for no weapon being set
 	if (custom == FALSE8 /*__NONE*/) {
-// custom must be in the current weapon set - bah, shouldn't have done it like this - its daft
+// custom must be in the current weapon set - bah, shouldn't have done it like this - it's daft
 		// rav (or equiverlant always come from pcp directory...
```

Comment:
should that be equivalent there?
Also missing a closing parenthesis in that comment

Reply:
Indeed. Done